README.md

```diff
@@ -380,6 +380,38 @@ Sent from the rover server to inform Mission Control of a single frame of a came
 - `camera` - the name of the camera: `mast|hand|wrist`
 - `data` - the raw h264 frame data, or `null` if no data is available
 
+## Camera Frame Request
+### Description
+Sent from Mission Control to instruct the rover server to send a Camera Frame Report.
```

Comment:
Super minor, but mention how this works with the report. I.e. "If camera specifies a valid camera stream, the rover will respond with a camera frame report containing the latest frame from that camera."